### ISSUE TITLE
A lot of `._` files in `volrender/sphere-cuboid`

### ISSUE THREAD

**Issue by rioj7** [mkvenkit/pp2e#1]
What is the need for the `._` files in `volrender/sphere-cuboid`. They are Mac OS specific and don't need to be in a repo.

**Comment by mkvenkit** [comment]
You're right. I need to clean that up... Hope the program is working for you, though.
